via/services/proxy_pdf.py

```diff
@@ -0,0 +1,55 @@
+from via.requests_tools import add_request_headers
```

Comment:
A toolbox approach for this it's not that bad, they are request headers we want to be used everywhere, I think the alternative would be a `Proxy` service with just that but just a functions seems better.